cf_xarray/accessor.py

```diff
@@ -850,6 +850,15 @@ def describe(self):
         print(text)
 
     def get_valid_keys(self) -> Set[str]:
+        
+        warnings.warn(
+                      """Now called `keys` and `get_valid_keys` will 
+                      be removed in a future version.""",
+                      DeprecationWarning,
+                      )
```

Comment:
```suggestion
                      )
                      
        return self.keys()
```

for backward compatibility

Reply:
Oops, forgot that part.

Reply:
did this in a separate commit since already changed the repo.